test/framework/convenience.go

```diff
@@ -96,6 +97,9 @@ func TryAddDefaultSchemes(scheme *runtime.Scheme) {
 	// Add the core CAPI scheme.
 	_ = clusterv1.AddToScheme(scheme)
 
+	// Add the experiments CAPI scheme.
+	_ = expv1.AddToScheme(scheme)
```

Comment:
Ah, this might be a good candidate for consuming the feature flag somehow...  I don't necessarily think that adding the flag is the right approach for this test, but might be good to conditionalize it somehow.

Reply:
I assumed that adding the types to the scheme shouldn't be a huge issue, if folks aren't using these type they'll live in the scheme but be ignored